examples/vercel-ai-chatbot/app/actions.ts

```diff
@@ -22,9 +18,9 @@ export async function getChats(userId?: string | null) {
       pipeline.hgetall(chat)
     }
 
-    const results = await pipeline.exec()
+    const results = await pipeline.exec<Chat[]>()
 
-    return results as Chat[]
+    return results
```

Comment:
we want to discourage "bad" developer patterns wherever possible, like `as` casting and type ignores